resources/views/components/nav-layer.blade.php

```diff
@@ -0,0 +1,57 @@
+@props(['id', 'children', 'title' => __('Menu'), 'hasParent' => false, 'tag' => 'form', 'parentUrl' => ''])
+@slots(['headerbutton'])
+
+<x-rapidez::slideover.mobile
+    :title="(string) $title"
+    :$id
+    :$hasParent
+    :$tag
+>
+    <x-slot:headerbutton>
+        <div class="absolute left-0 top-1/2 -translate-y-1/2 cursor-pointer text-white">
+            @include('rapidez-statamic::navigation.header-button')
+        </div>
+    </x-slot:headerbutton>
+    <div class="flex w-full flex-1 flex-col bg-inactive-100">
+        <ul class="mt-5 flex flex-col divide-y border-y bg-white">
+            @if ($hasParent && $parentUrl)
+                <li>
+                    <a href="{{ $parentUrl }}" class="normal flex items-center justify-between p-5 font-semibold">
+                        @lang('Go to :item', ['item' => strtolower($title)])
+                    </a>
+                </li>
+            @endif
+            @foreach ($children ?: [] as $child)
+                @php
+                    $url = $child['url'];
+                    if (isset($child['linked_category'])) {
```

Comment:
I don't think `linked_category` is a default field, can you add the blueprint?

Reply:
It actually is **not**. on all our projects this is a field on the navigation blueprint, not on the category itself.